templates/java-gradle/src/main/java/my/example/Greeter.java

```diff
@@ -25,7 +25,7 @@ public class Greeter {
 
   @Handler
   public String greet(Context ctx, String greeting) {
-    return greeting;
+    return "Hello " + greeting;
```

Comment:
This would respond Hello Hi to the current quickstart request, let's leave it at the previous. Anyway I am changing the quickstarts

Reply:
ok